mungegithub/publisher/deployment.yaml

```diff
@@ -10,6 +10,25 @@ spec:
         app: publisher
         target-repo: @@
     spec:
+      initContainers:
+      - name: initialize-repos
+        command:
+        - /bin/bash
+        - -xc
+        - "ORG=$(grep organization: /etc/munge-config/config | cut -d' ' -f2) /publish_scripts/initialize_repos.sh"
+        - 2>&1
+        image: gcr.io/k8s-testimages/publisher:2016-05-24-86f86cd
+        imagePullPolicy: Always
+        resources:
+          requests:
+            cpu: 200m
```

Comment:
Is 200m too small? How long does the initialization cost you?

Reply:
Changed it to the main container values.

Reply:
Am pretty sure though that it's more IO bound usually.